app/controllers/curated_clusters_controller.rb

```diff
@@ -19,6 +19,12 @@ def create
     redirect_to "/users/#{@user.username}/curated_clusters"
   end
 
+  def addarticle
+    cluster = CuratedCluster.find(params["cluster_id"])
+    article = Article.find(params["article_id"])
+    cluster.articles << article
+  end
+
```

Comment:
this duplicate code needs to be removed--i dont think it does anything rn

Reply:
On it! I'll take this out

Reply:
Updated in latest commit